src/redux/configureStore.js

```diff
@@ -1,16 +1,14 @@
-import React, { Component } from 'react';
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
```

Comment:
- [ ] You currently do not have `redux-logger` in your `package.json` file. Kindly install it

  ![redux-logger](https://user-images.githubusercontent.com/36542986/173551773-d2be329e-c9fe-4cb4-a973-e224b4848a7d.jpg)